### PR TITLE
fix: dev app deeplinking

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
we need deeplinking on dev app to do redirect testing. Also, added domain name in env variable. (also added from vercel dashboard)

# Test Plan
- Test deeplinking with dev app.
- Make sure to copy `NEXT_PUBLIC_WEBSITE_DOMAIN` the variable in .env file in your local next app.

### CODE REVIEW

File: apps/expo/.env.development [hunk 1]
```diff
@@ -25,3 +25,4 @@ SENTRY_SERVER_ENVIRONMENT=local
 SENTRY_SERVER_RELEASE_ENVIRONMENT=local
 
 NEXT_PUBLIC_META_SINGLE_EDITION_MINTABLE_CREATOR=0x50C001A33Caa446c8b84C489F371F77754F41024
+NEXT_PUBLIC_WEBSITE_DOMAIN=dev.showtime.xyz
```
Comment (intergalacticspacehighway) [inline apps/expo/.env.development:28]: we need to point `dev` CNAME record to vercel. cc - @axeldelafosse do you have access?


Screenshot 2022-10-03 at 5 02 25 PM

**Review by alantoa — APPROVED** [review]
LGTM ✨

File: apps/expo/.env.development (hunk 1 above)
Comment (axeldelafosse) [reply apps/expo/.env.development:28]: Not anymore. You should ask Alex for the credz

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Pull request merged**